kafka-connect-neo4j/src/main/kotlin/streams/kafka/connect/sink/Neo4jSinkConnector.kt

```diff
@@ -36,7 +36,7 @@ class Neo4jSinkConnector: SinkConnector() {
         val migratedConfig = ConfigurationMigrator(settings).migrateToV51()
         val mapper = ObjectMapper()
         val jsonConfig = mapper.writeValueAsString(migratedConfig)
-        log.info("Migrated Sink configuration to v5.1 connector format: {}", jsonConfig)
+        log.info("Migrated Neo4j Sink Connector '{}' configuration to v5.1 connector format: {}", settings["name"], jsonConfig)
```

Comment:
I think we can improve this message a bit, something along the lines of;

`The migrated settings for 5.1 version of Neo4j Sink Connector '{}' is: {}`

I'm not sure how it is shown in the logs but should we also wrap the configuration into backticks or quotes?